sympy/stats/error_prop.py

```diff
@@ -0,0 +1,74 @@
+"""Tools for arithmetic error propogation."""
+from __future__ import print_function, division
+from itertools import repeat
+
+from sympy import Symbol, symbols, Add, Mul, simplify, Pow, exp
```

Comment:
@certik I use the `flycheck-pyflakes` plugin for emacs. pyflakes in particular is the preferable linter because it only shows logical errors (unlike flake8 or pylint, which also show style errors, which I often tend to disagree with).